vulture/core.py

```diff
@@ -415,6 +420,33 @@ def visit(self, node):
             visitor(node)
         return self.generic_visit(node)
 
+    def _handle_ast_list(self, ast_list, parent_node):
+        for index, node in enumerate(ast_list):
+            if isinstance(node, ast.Return):
+                try:
+                    first_unreachable_node = ast_list[index + 1]
+                except IndexError:
+                    pass
```

Comment:
let's `continue` here and remove the `else` clause. Then the code can be moved to the left by four spaces.